src/content/en/showcase/case-study/beyond-the-rack.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: showcase/casestudy
+title: "Beyond the Rack"
+subtitle: "Beyond the Rack re-engages users on the mobile web, increasing revenue per visit by 26% with push notifications"
+description: "&quot;<span style='font-style: italic;'>Push notifications allowed us to bring one of the most compelling capabilities from our native app to our mobile site. We see a direct 20% click through rate from push notifications—having another channel to reach our users is a game changer.</span>&quot; <br><br> &mdash;  Richard Cohene, VP Marketing, Beyond the Rack"
```

Comment:
Sorry, I just looked at your code in the comment :) < i > is not the way we should be italicised as it is topical subtext.... thinking about it < em > is probably the best to use.